nhttp/ascii.go

```diff
@@ -66,6 +66,25 @@ func lower(b byte) byte {
 	return b
 }
 
+// Checks if the given string is composed exclusively of uppercase alphanumeric characters.
+func IsUppercaseAlphanumeric(str string) bool {
```

Comment:
This function should probably go in `common/strings.go`. This directory is a modified version of Go's HTTP library.

Reply:
Resolved in commit [333c797](https://github.com/WiiLink24/wfc-server/pull/20/commits/333c79701d2a04603be12dd7852679fcd4353729).